PR#3414 (feat: add doc for seat-based billing plans) follow-up (#3435)
clerkinc/clerk-docs: docs/guides/billing/seat-based-plans.mdx

This makes it possible to tailor Plans to Organizations based on size, or to offer pricing that scales with growth.

## Create a seat-based Plan

- **Seat limits** determine the maximum number of seats an Organization may have.

> - **Per-seat pricing** determines how much an Organization is charged for each seat.

Reviewer: I feel like we need an intro sentence before the bullet pts. Sth along the lines of: 

A seat-based Plan consists of configuring seat limits and per-seat pricing: 

Then have the explanation of each? @alexisintech

Author: yeah that works!

> Seat limits can be configured on their own or alongside per-seat pricing. If you want to do per-seat pricing without a seat limit, you can set the seat limit to unlimited.

Reviewer: ```suggestion
Seat limits can be configured on their own or alongside per-seat pricing. If you want to do per-seat pricing without a seat limit, you can set the seat limit to **unlimited members**.
```

To keep the vocabulary of the Dashboard.

1. In the Clerk Dashboard, navigate to the [**New Organization Plan**](https://dashboard.clerk.com/~/billing/plans/new/org) page.
1. Enable **Seat-based** and choose how many seats the Plan allows:
   - To cap the number of seats, select **Custom limit** and enter the desired value.
   - To allow an unlimited number of seats, select **Unlimited members**.
   > [!NOTE]
   > The [B2B Authentication add-on](https://clerk.com/pricing){{ target: '_blank' }} is required to set a custom limit greater than 20 seats or to allow unlimited members.
1. If you want to charge Organizations based on the number of purchased seats, enable **Per-seat fee** and configure the pricing:
   - **Cost per member seat monthly**: Enter the price per seat.

>    - **Included seats**: Enter the number of seats that are included in the Plan for free.

Reviewer: ```suggestion
   - **Included seats**: Enter how many seats are included in the Plan before per-seat charges apply.
```

Just as a suggestion, as to me, this sentence (**Enter the number of seats that are included in the Plan for free.**) reads as if the action of entering the nb of seats is free.
